### ISSUE TITLE
HELP!!! HOW TO MAKE THE CONDITION!!!

### ISSUE THREAD

**Issue by ZDEGLET** [binary-com/binary-bot#682]
PURCHASE ALWAYS UP.
1 CONTRACT CALL -  RESULT Win
2 CONTRACT CALL -  RESULT Win
3 CONTRACT CALL -  RESULT Win
4 CONTRACT CALL -  RESULT Win
5 CONTRACT CALL -  RESULT Win
6 CONTRACT CALL -  RESULT Loss # 1
7 CONTRACT CALL -  RESULT Loss **# 2**
8 CONTRACT CALL -  RESULT Loss **# 3**
9 CONTRACT CALL -  RESULT Loss **# 4**
10 CONTRACT **# PUT** -  RESULT is not important
PURCHASE ALWAYS UP
11 CONTRACT CALL
12 CONTRACT CALL
13 CONTRACT CALL
9 CONTRACT CALL   .......................................................................

**Comment by Eshhh** [comment]
![image](https://user-images.githubusercontent.com/33750313/39398727-7d796440-4b2f-11e8-95e5-3bceff20b19a.png)

**Comment by ZDEGLET** [comment]
@Eshhh after 4 losses 1 time down then again always up

**Comment by Eshhh** [comment]
![image](https://user-images.githubusercontent.com/33750313/39398913-842639e6-4b32-11e8-87d2-4f9bf308c9e0.png)

[Call but Put after 4 loss.zip](https://github.com/binary-com/binary-bot/files/1957930/Call.but.Put.after.4.loss.zip)

**Comment by ZDEGLET** [comment]
Thank you so much!!!
God grant you health!!!
Now I'll see how it will work.

**Comment by tymu** [comment]
@Eshhh the code is looking for overall number of losses, for ex: if it was 3 losses 1 win and 1 loss = 4 loss?

**Comment by Eshhh** [comment]
@tymu No its looking for 4 consecutive losses.

**Comment by ZDEGLET** [comment]
after 4 losses 1 time down, then again always up. when it continues upwards, after 4 losses 1 time down does not work for some reason.

**Comment by Eshhh** [comment]
@ZDEGLET the logic also considers the result of the put trade too. So if you want 4 consecutive loss only on the call then try this..

![image](https://user-images.githubusercontent.com/33750313/39539989-fb76db52-4e5e-11e8-8a39-48a7d8c8bc37.png)

[Call but Put after 4 loss V2.zip](https://github.com/binary-com/binary-bot/files/1968340/Call.but.Put.after.4.loss.V2.zip)

**Comment by Jonni81** [comment]
@ZDEGLET Что именно ты хочешь собрать?

**Comment by ZDEGLET** [comment]
@Jonni81 ДА ТАК ГОНЯЮ ОДИН БОТ ХОЧУ ПОСМОТРЕТЬ КАК С ЭТОЙ СТРАТЕГИЕЙ БУДЕТ РАБОТАТЬ

**Comment by ZDEGLET** [comment]
@tymu О ЧЕМ ВЫ ХОТИТЕ СО МНОЙ ПОБАЛАКАТЬ?

**Comment by Jonni81** [comment]
@ZDEGLET ты его без мартина гоняешь? только я не понимаю на чем он у тебя основывается

**Comment by Eshhh** [comment]
@tymu whatsup..?

**Comment by ZDEGLET** [comment]
@Jonni81 ПОГОНЯЮ ПОТОМ НАПИШУ ЕСЛИ ВСЕ ОКЕЙ БУДЕТ. Я ЕГО С МАРТИНОМ ГОНЯЮ. ПОСМОТРЮ ЧТО ПОЛУЧИТСЯ.
БЕЗ МАРТИНА Я НЕ ВИДЕЛ НИ ОДНОГО БОТА.

**Comment by Jonni81** [comment]
@ZDEGLET Я не понимаю в чем прикол проиграть 4 сделки и открыть 5 в противоположную сторону

**Comment by ZDEGLET** [comment]
@Jonni81 ИЗ ВСЕГО ЧТО ЗДЕСЬ ПИШЕТСЯ И ВЫСТАВЛЯЕТСЯ Я НОРМАЛЬНОГО И ПРИКОЛЬНОГО НЕХЕРА НЕ ВИДЕЛ. ВСЕ БОТЫ 50 НА 50 И В КОНЕЧНОМ ИТОГЕ ПРОСТО СЛИВ ПО ПОЛНОЙ.

**Comment by Jonni81** [comment]
@ZDEGLET можем вместе попробовать этого довести до ума,если хочешь

**Comment by ZDEGLET** [comment]
@Jonni81 ТОГДА ИЗ ЭТОЙ ТЕМЫ НУЖНО 3 ПРОИГРАЛ 4 ПРОТИВОПОЛОЖНАЯ, ПОТОМ 1 ПРОИГ. 1 ПРОТИВОПОЛОЖ, ПОТОМ 2 ПРОИГРАЛ 1 ПРОТИВОПОЛОЖ., ОБЪЕДИНИТЬ НЕ МОГУ ЭТИ ТРИ ВАРИАНТА.

**Comment by Jonni81** [comment]
@ZDEGLET kazarinov.treiding@yandex.ru скинь я взгляну

**Comment by ZDEGLET** [comment]
@Jonni81 ТАК ОН НА ВЕРХУ СДЕЛАН ТОЛЬКО В ОДНОМ ВАРИАНТЕ.

**Comment by Jonni81** [comment]
ок

**Comment by ZDEGLET** [comment]
@Jonni81 ТЫ ХОЧЕШЬ СКАЗАТЬ ЧТО СДЕЛАЕШЬ. ЕСЛИ СДЕЛАЕШЬ ТЫ ПЕРЕЦ. Я НЕКОТОРЫХ МОМЕНТОВ В ЭТИХ БОТАХ НЕ ДОГОНЯЮ СУТИ.

**Comment by Jonni81** [comment]
@ZDEGLET Сегодня посижу. Я думаю соберем

**Comment by Jonni81** [comment]
@ZDEGLET Скинь мне электронку я тебе отправлю, а то не архивируется нихрена

**Comment by ZDEGLET** [comment]
@Jonni81 zdeglet@gmail.com

**Comment by okovalov** [comment]
@Jonni81 @ZDEGLET ребята, ну а какая здесь стратегия-то? 4 loss потом PUT - и так по кругу? а какие-то расчеты , индикаторы? я тоже не понимаю смысла.

**Comment by ZDEGLET** [comment]
@Jonni81 zdeglet@mail.ru

**Comment by gokula-krishna-dev** [comment]
We're closing this issue due to inactivity, you're welcome to open it again if you still haven't got your answer.
